packages/orbit-components/src/Truncate/index.tsx

```diff
@@ -1,32 +1,29 @@
 "use client";
 
 import * as React from "react";
-import styled from "styled-components";
+import cx from "clsx";
 
 import type { Props } from "./types";
 
-const StyledTruncate = styled.div<{ maxWidth?: string }>`
-  ${({ maxWidth }) => `
-    min-width: 0;
-    flex: 0 1 ${maxWidth === "none" ? "100%" : maxWidth};
-    max-width: ${maxWidth};
-  `};
-`;
-
-const StyledTruncateContent = styled.div`
-  &,
-  .orbit-text,
-  .orbit-heading {
-    white-space: nowrap;
-    overflow: hidden;
-    text-overflow: ellipsis;
-  }
-`;
-
 const Truncate = ({ children, maxWidth = "100%", dataTest }: Props) => (
-  <StyledTruncate maxWidth={maxWidth} data-test={dataTest}>
-    <StyledTruncateContent>{children}</StyledTruncateContent>
-  </StyledTruncate>
+  <div
+    data-test={dataTest}
+    className={cx(
+      "min-w-0 max-w-[--max-width] shrink grow-0",
```

Comment:
TIL that this actually works for CSS vars and we don't need `var(…)`. To me I would always write it as `max-w-[var(--max-width)]`, which is how we have it on the rest of the components… but apparently it works fine so I am not against it at all.

Reply:
I guess that either I discovered this feature by accident or I saw it somewhere, I don't remember 😅.  I ran a quick search on Tailwind and this behavior [is documented](https://tailwindcss.com/docs/adding-custom-styles#using-arbitrary-values) so we are safe!


<img width="764" alt="image" src="https://github.com/kiwicom/orbit/assets/8863238/1ec9bfd4-ad9f-4aea-9a4c-01428dccfc01">